lib/streamlit/elements/lib/column_types.py

```diff
@@ -525,14 +525,13 @@ def LinkColumn(
 
         * A regular expression (JS flavor, detected by usage of parentheses)
           to extract a part of the URL via a capture group, e.g. ``"https://(.*?)\.streamlit\.app"``
-          to extract the display text "foo" from the URL "https://foo.streamlit.app".
+          to extract the display text "foo" from the URL "\https://foo.streamlit.app".
```

Comment:
What's the reason for the slash here?

Reply:
It makes it render in plain text so it doesn't become a clickable link in the docs site. foo.streamlit.app isn't a real app. In fact, it might be better to write it as foo.example.com since someone could theoretically grab foo on Community Cloud.